Makefile

```diff
@@ -0,0 +1,55 @@
+COLOUR_GREEN=\033[0;32m
+COLOUR_RED=\033[0;31m
+COLOUR_BLUE=\033[0;34m
+COLOUR_END=\033[0m
+
+.DEFAULT_GOAL:=help
+
+SHELL := /bin/bash
+GENERAL_LIB_LOCATION := $(shell pip show rados | grep -oP "(?<=Location: ).*")
+LIBRADOS_LOCATION := $(shell find "${GENERAL_LIB_LOCATION}" -name 'rados*.so' -print)
+LIBRADOS_EGGINFO_LOCATION := $(shell find "${GENERAL_LIB_LOCATION}" -name 'rados*.egg-info' -print)
+LIBRADOS_NAME := $(shell basename "${LIBRADOS_LOCATION}")
+LIBRADOS_EGGINFO_NAME := $(shell basename "${LIBRADOS_EGGINFO_LOCATION}")
+
+## checking if docker, or podman should be used. Podman is preferred.
+ifeq ($(shell command -v podman 2> /dev/null),)
+	CONTAINERCMD=docker
+else
+	CONTAINERCMD=podman
+endif
+
+
+.PHONY: help
+help: ## Display this help message
+	@echo -e '${COLOUR_RED}Usage: make <command>${COLOUR_END}'
+	@cat $(MAKEFILE_LIST) | grep '^[a-zA-Z]'  | \
+	    awk -F ':.*?## ' 'NF==2 {printf "  %-26s%s\n\n", $$1, "${COLOUR_GREEN}"$$2"${COLOUR_END}"}'
+
+.PHONY: setup
+setup: ## Setting up the docker-compose environment
+	${MAKE} setup-venv
+	
+setup-venv:
+	python -m venv ./.venv
```

Comment:
You may want to use `python -m venv --system-site-packages --symlinks` as in the `Dockerfile`. In that case you do not have to symlink files manually.

Reply:
I think using --system-site-packages is not a good solution, especially in dev environment, as pip freeze will then list all system packages and you can't really define, what version of a site-package is loaded, if all system site-packages are included in venv

Reply:
ok for makefile I opened a separate mr here: https://github.com/SovereignCloudStack/rookify/pull/29